frontend/src/app/topic-drafts/topic-drafts-component/topic-drafts.component.ts

```diff
@@ -63,9 +66,4 @@ export class SubmittedTopicDraftsComponent implements OnInit {
         this.rowData.data = topicDrafts;
       });
   }
-
-  navigateToCompanies(): void {
-    this.router.navigate(['companies'])
-      .catch();
-  }
```

Comment:
wieso keine navigation mehr?

Reply:
Die Navigation wurde in das html-template verschoben durch ein [routerLink] Attribut